docs/scaling/secrets.md

```diff
@@ -191,9 +191,9 @@ AWS Secrets Manager console) as a source:
 
 ### Accessing secrets in Azure
 
-Azure Key Vault is an account specific service, managed via Azure
-Resource Manager. Currently, only `Secret` object types are supported. You can specify 
-secrets in the `sources` list or dictionary object as shown below. 
+Starting 2.11.16 Metaflow supports Azure Key Vault as a secrets backend. Azure Key Vault is an 
```

Comment:
Do we need to add versions? We haven't added versions in docs usually (if there are any we should strip them out)

Reply:
removed.